### PR TITLE
Fix Python install in workflows

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: core/src/test/java/org/mapfish/print/processor/map/CreateMapProcessorWmtsBufferTest.java [hunk 1]
```diff
@@ -76,7 +76,7 @@ public void testExecute() throws Exception {
     List<URI> layerGraphics = (List<URI>) values.getObject("layerGraphics", List.class);
     assertEquals(1, layerGraphics.size());
 
-    new ImageSimilarity(new File(layerGraphics.get(0)))
-        .assertSimilarity(getFile(BASE_DIR + "/expectedSimpleImage.png"), 0);
+    new ImageSimilarity(getFile(BASE_DIR + "/expectedSimpleImage.png"))
```
Comment (sebr72) [inline core/src/test/java/org/mapfish/print/processor/map/CreateMapProcessorWmtsBufferTest.java:79]: Good idea. Thanks
Have you double checked if this was the only place where we called it wrong ?

**Review by sebr72 — CHANGES_REQUESTED** [review]
I am not sure why the images were regenerated.
But one is definitely not working.

File: core/src/test/java/org/mapfish/print/processor/map/CreateMapProcessorWmtsBufferTest.java (hunk 1 above)
Comment (sbrunner) [reply core/src/test/java/org/mapfish/print/processor/map/CreateMapProcessorWmtsBufferTest.java]: As I see yes :-)

**Comment by sbrunner** [comment]
> I am not sure why the images were regenerated.

Oups, I get an extra commit.

**Review by sebr72 — APPROVED** [review]

**Pull request merged**